addons/project/static/src/views/project_task_graph/project_task_graph_model.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+
+import { GraphModel } from "@web/views/graph/graph_model";
+
+export class ProjectTaskGraphModel extends GraphModel {
+    _getDefaultFilterLabel(fieldName) {
```

Comment:
```suggestion
    _getDefaultFilterLabel(fields) {
```
It isn't a fieldName, but rather a list of fields, right? (because you filter)

Reply:
Done Thanks :+1: